src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/tabsontop/_cq_template.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
     jcr:primaryType="nt:unstructured"
-    jcr:title="Tabs on top"
+    jcr:title="Horizontal Tabs"
```

Comment:
we should also include two panels by default in tabs just like in accordion.

Reply:
Done